packs/fastly/dashboards/fastly-cdn.json

```diff
@@ -0,0 +1,228 @@
+{
+  "name": "Fastly",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
this `permissions` line will have to be removed manually for now
and all of the `description` fields should be an empty string "" instead of null
(otherwise the schema validation check will fail, these are some known issues we will improve but have to be fixed manually now)
Thanks!

Reply:
Done